tests/abortmultipartupload_test.py

```diff
@@ -0,0 +1,58 @@
+from unittest import *
+import inspect
+import os
+import json
+from libs.execute import *
+from libs.command import *
+from libs.utility import *
```

Comment:
Replace the use of `*` with the things needed from the module.

Alternatively, you can import the module and qualify each thing from it with the proper prefix (e.g. `libs.execute.assert_command()`).

If other tests are using `*`, open an issue to replace those. We can handle them later.

Reply:
I went ahead and fixed these wildcard imports.  Wrote an issue (#135) for the others and went ahead and made the change for the others in a separate commit.  Test cases pass.

Reply:
Nice. As soon as those changes are pushed, we'll look at getting this work merged.